### PR TITLE
Source Google Ads: add `segments.device` and remove `metrics.historical_quality_score` from `keyword_report`

### PR DESCRIPTION
**Pull request by LuliRib**
adding segments.date field to keyword_report. 

Tests succedded + Google ads response in pr#20005

### CODE REVIEW

**Comment by LuliRib** [comment]
@marcosmarxm Good morning! This is thenew branch with the google ads field added to keyword_report schema.
As it´s a new branch, the tests don´t appear here.. but in #20005

**Comment by LuliRib** [comment]
@sh4sh, hi, maybe you can help me. I was asked to open this new branch. The google ads test passed ok in the other branch..

**Comment by marcosmarxm** [comment]
@grubberr can you validate this change removing one field from Google Ads to add other?

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json [hunk 1]
```diff
@@ -57,17 +57,17 @@
     "ad_group_criterion.keyword.match_type": {
       "type": ["null", "string"],
       "enum": ["BROAD", "EXACT", "PHRASE", "UNKNOWN", "UNSPECIFIED"]
-    },
-    "metrics.historical_quality_score": {
-      "type": ["null", "integer"]
-    },
+    }
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json:60]: @LuliRib it is missing a comman and breaking the tests.
```suggestion
    },
```

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

**Comment by LuliRib** [comment]
Changed. Can you pls re-test?

**Comment by marcosmarxm** [comment]
@grubberr can you check this change for Google Ads?

**Comment by LuliRib** [comment]
@marcosmarxm hi, is there any way to move forward with it?

**Comment by LuliRib** [comment]
@marcosmarxm Hi

**Comment by LuliRib** [comment]
Hi, any way to move forward with it?

**Comment by marcosmarxm** [comment]
Hello :wave::skin-tone-2: and thank you for your contribution!

Airbyte has instituted a code freeze between 19 and 30 December, to make sure there are no disruptions during the holidays. 
Because of this, reviewing and merging your contribution may take longer than usual. 
We apologize for the delay, but we want everyone to have a quiet and happy holiday.

If you have any questions or need further clarification, please don't hesitate to ping via Slack.

**Comment by sajarin** [comment]
@grubberr can you approve this PR, it's a schema update for the Google Ads connector.

**Review by sajarin — APPROVED** [review]

**Review by lazebnyi — CHANGES_REQUESTED** [review]
@LuliRib @marcosmarxm I'm having trouble understanding why we need to replace `metrics.historical_quality_score` with `segments.device` and trigger a breacking changes. If we require additional fields that are incompatible with the fields already in the schema, we can use the `custom_queries` field in the configuration to obtain what we need.

**Comment by marcosmarxm** [comment]
@lazebnyi overall I think segments.device is more valuable info than metric quality score. But @LuliRib probably has a better argument than mine.

**Comment by LuliRib** [comment]
Hi all, the problem is that those 2 fields are not compatible (can check it in the query builder).
I think the metrics.device is more valuable.


Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request closed without merging**